lib/runtime-c-api/src/lib.rs

```diff
@@ -17,23 +17,23 @@ use wasmer_runtime_core::module::{ExportIndex, ImportName};
 use wasmer_runtime_core::types::{ElementType, FuncSig, MemoryDescriptor, TableDescriptor, Type};
 use wasmer_runtime_core::units::{Bytes, Pages};
 
-#[allow(non_camel_case_types)]
-pub struct wasmer_module_t();
+#[repr(C)]
```

Comment:
`repr(C)` implies `no_mangle` and `allow(non_camel_case_types)` as my tests shown. Not sure, but it has removed several warnings.

Reply:
👍, also when changes are made that may affect the header files you can run `make capi` from the root project. That and running tests is described in this projects `README`. Please run `make capi` for this PR if you  haven’t already.

Reply:
Done. I think we have an issue :-).